src/internal/kopia/base_finder.go

```diff
@@ -55,6 +55,20 @@ type BackupBase struct {
 	Reasons []identity.Reasoner
 }
 
+func (bb BackupBase) GetReasons() []identity.Reasoner {
+	return bb.Reasons
+}
```

Comment:
since we're now implementing a function for this, would you please go through and make `bb.Reasons` unexported and replace calls to it with this function?

Reply:
It's being set by external packages in some places.  We can tackle that as tech debt another time.